### ISSUE TITLE
a feature request (inter-chromosomal structural variants) for mason_variator

### ISSUE THREAD

**Issue by abysslover** [seqan/seqan#1308]
I have used mason_variator to simulate various types of structural variants.
The problem is that I need simulations for inter-chromosomal translocations and indels given two chromosome A and B.
For instance, some sequences are selected from A and moved to B and information about this mutation would be like A[position with respect to the original A chromosome, length], B[position with respect to the original B chromosome, length].

Thanks in advance.

**Comment by h-2** [comment]
@holtgrewe can you have a look? Thank you.

**Comment by holtgrewe** [comment]
I think that this is a very important point. Its impact on the Mason could would be quite large and I will have to look into this and see what has to be changed to make it work.

**Comment by teng-gao** [comment]
Did this feature ever get implemented?

**Comment by holtgrewe** [comment]
No, it did not. I will probably not have the time to go for this.
